src/claims/models/claim-scope.model.ts

```diff
@@ -0,0 +1,30 @@
+/* istanbul ignore file */
+
+import { ApiProperty, ApiPropertyOptional } from '@nestjs/swagger';
+import { Scope } from '@polymeshassociation/polymesh-sdk/types';
+
+export class ClaimScopeModel {
+  @ApiProperty({
+    description: 'The scope that has been assigned to Identity',
+    example: {
+      type: 'Identity',
+      value: '0x6'.padEnd(66, '1a'),
+    },
+  })
+  readonly scope: Scope | null;
```

Comment:
IIRC we have a ScopeModel which should be used here.

Also add nullable: true, define type field as well